### PR TITLE
Dropping support for Node.js 6 and npm 3

### PR DESCRIPTION
**Pull request by dgaubert**
- Drop support for:
  - Node.js 6
  - npm 3 and npm-shrinkwrap.json file
- Remove scripts to sanitize npm-shrinkwrap.json
- Update docs and NEWS

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -209,8 +210,8 @@
     "lint:css": "stylelint './app/assets/stylesheets/**/*.scss'",
     "lint:fix": "eslint . --fix",
     "bump": "npm --no-git-tag-version version prerelease --preid=assets",
-    "postversion": "git add package.json npm-shrinkwrap.json && git commit -m 'Bump assets version' && git push origin master",
-    "update-internal-deps": "rm -rf node_modules && npm install --production --no-optional --no-shrinkwrap && npm dedupe && npm prune && npm shrinkwrap && npm run fix-shrinkwrap-protocol && npm install",
+    "postversion": "git add package.json package-lock.json && git commit -m 'Bump assets version' && git push origin master",
+    "update-internal-deps": "rm -rf node_modules && npm install && npm install",
```
Comment (VictorVelarde) [inline package.json]: why do we have two `npm install` here?

Comment (dgaubert) [reply package.json]: typo

**Comment by dgaubert** [comment]
Should I merge or you want me to pass some acceptance?

**Comment by jesusbotella** [comment]
GitHub says that the branch has conflicts :trollface:. Let's fix them and deploy to staging to test that everything works properly, just in case.

**Comment by dgaubert** [comment]
> Let's fix them and deploy to staging to test that everything works properly, just in case.

✅ Done!

**Pull request merged**